lib_device_control/host/device_access_usb.c

```diff
@@ -213,14 +214,14 @@ control_ret_t control_init_usb(int vendor_id, int product_id, int interface_num)
 
   int r = usb_set_configuration(devh, 1);
   if (r < 0) {
-    fprintf(stderr, "Error setting config 1\n");
+    PRINT_ERROR("Error setting config 1\n");
     usb_close(devh);
     return CONTROL_ERROR;
   }
 
   r = usb_claim_interface(devh, interface_num);
   if (r < 0) {
-    fprintf(stderr, "Error claiming interface %d %d\n", interface_num, r);
+    PRINT_ERROR("Error claiming interface %d %d\n", interface_num, r);
```

Comment:
As "error" would otherwise be repeated twice, I would substitute "failed to claim interface" for "Error claiming interface".